docs/src/content/docs/fr/resources/plugins.mdx

```diff
@@ -38,6 +38,26 @@ Les [modules d'extension](/fr/reference/plugins/) peuvent personnaliser la confi
 		title="starlight-typedoc"
 		description="Générer des pages Starlight à partir de TypeScript en utilisant TypeDoc."
 	/>
+  <LinkCard
+    href="https://github.com/HiDeoo/starlight-blog"
+    title="starlight-blog"
+    description="Ajouter un blogue à votre site de documentation."
+  />
+  <LinkCard
+    href="https://github.com/HiDeoo/starlight-openapi"
+    title="starlight-openapi"
+    description="Créer des pages de documentation à partir de spécifications OpenAPI/Swagger."
+  />
+  <LinkCard
+    href="https://github.com/HiDeoo/starlight-obsidian"
+    title="starlight-obsidian"
+    description="Publier vos Vaults d'Obsidian sur votre site Starlight."
+  />
+  <LinkCard
+    href="https://astro-ghostcms.xyz/intro/starlight/install/"
+    title="starlight-ghostcms"
+    description="Ajoutez vos articles de blog GhostCMS à votre documentation Starlight"
```

Comment:
```suggestion
    description="Ajoutez vos articles de blogue GhostCMS à votre documentation Starlight"
```

Just noticed we used "blogue" a few lines above so we should properly translate both.